### PR TITLE
&1 attributes reach threshold of &2

### PR DESCRIPTION
**Pull request by lucasborin**

### CODE REVIEW

**Review by estevao-schultz-neto-SAP — CHANGES_REQUESTED** [review]
"reaching"

File: src/checks/y_check_number_methods.clas.abap [hunk 1]
```diff
@@ -49,7 +49,7 @@ CLASS Y_CHECK_NUMBER_METHODS IMPLEMENTATION.
     settings-threshold = 20.
     settings-documentation = |{ c_docs_path-checks }number-methods.md|.
 
-    set_check_message( '&1 methods, exceeding threshold of &2' ).
+    set_check_message( '&1 methods attributes reaching threshold of &2' ).
```
Comment (lucasborin) [inline src/checks/y_check_number_methods.clas.abap:52]: number of methods, not attributes

File: src/checks/y_check_num_output_parameter.clas.abap [hunk 2]
```diff
@@ -21,7 +21,7 @@ CLASS Y_CHECK_NUM_OUTPUT_PARAMETER IMPLEMENTATION.
     settings-threshold = 2.
     settings-documentation = |{ c_docs_path-checks }number-output-parameter.md|.
 
-    set_check_message( 'Too many output parameters!' ).
+    set_check_message( '&1 attributes reaching threshold of &2' ).
```
Comment (lucasborin) [inline src/checks/y_check_num_output_parameter.clas.abap:24]: number of output parameters, not attributes

**Comment by lucasborin** [comment]
@estevao-schultz-neto-SAP : Could you please check it agian?

**Comment by estevao-schultz-neto-SAP** [comment]
fine!!

**Pull request merged**